db/migrate/20200224152354_remove_discipline_from_team_members.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveDisciplineFromTeamMembers < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :team_members, :discipline
+  end
```

Comment:
May I suggest doing a rollback instead of a remove_column? 
You can do this by the following:
- First you will need to add the `type` into this migration (:string - on line 3)
- The run a `rails db:rollback STEP=2` to rollback 2 migrations steps.
- This will take you to migration `add_metadata_fields_to_project`
- Delete the add and remove column migrations

Reply:
I need to do more reading on the db rollback as I'm not familiar enough. I'll do it in the afternoon.

Just to make sure I understood to avoid going back and forth with this again... You want me to delete the `role` column and add the `discipline` column again by doing the `db:rollback` (and then substitute all references to role for discipline). Also, for clarification purposes, are you sure you mean the `add_metadata_fields_to_project` migration file as the one I should change? I think the `role` column lives in `add_meta_fields_to_team_member` but as I said, I'm not familiar enough with rolling back the database, so I just want to double check. Thank you!